docker-compose.yml

```diff
@@ -22,7 +22,9 @@ services:
   web:
     build:
       context: .
-      dockerfile: Dockerfile
+      target: web
+      cache_from:
+        - "${REGISTRY_HOST}${REGISTRY_URI}/oral-history:${TAG}"
```

Comment:
Adding to help speed up the build process. Not sure how to tell if it has or not?